Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM heroku/heroku:16-build as base
 
-ENV PHP_BUILDPACK_VERSION v148
+ENV PHP_BUILDPACK_VERSION v178
```

Comment:
I was running into an error building the image locally, in previous PR #800 we had to update the buildpack version, so I did the same, to the latest [v178](https://github.com/heroku/heroku-buildpack-php/tree/v178)
```
-----> Bootstrapping...

 !     ERROR: Failed to download minimal PHP for bootstrapping!
 !     
 !     This is most likely a temporary internal error. If the problem
 !     persists, make sure that you are not running a custom or forked
 !     version of the Heroku PHP buildpack which may need updating.

ERROR: Service 'web' failed to build: The command '/bin/sh -c /tmp/buildpack/php/bin/compile /app /tmp/build_cache /tmp/env' returned a non-zero code: 1
```